src/custom/utils/appData.ts

```diff
@@ -18,20 +18,18 @@ export async function buildAppData({
   appCode,
   orderClass,
 }: BuildAppDataParams) {
-  const sdk = COW_SDK[chainId]
-
   const referrerParams =
     referrerAccount && chainId === SupportedChainId.MAINNET ? { address: referrerAccount } : undefined
 
   const quoteParams = { slippageBips }
   const orderClassParams = { orderClass }
 
-  const doc = sdk.metadataApi.generateAppDataDoc({
+  const doc = await metadataApiSDK.generateAppDataDoc({
```

Comment:
Shouldn't this come from appData?

Reply:
Yes, and it does. See `cow-react/cowSdk.ts`